docs/discovery.md

```diff
@@ -19,7 +19,7 @@ composer require "php-http/discovery"
 
 ## HTTP Client Discovery
 
-This type of discovery finds installed HTTP Clients.
+This type of discovery finds a HTTPClient implementation.
```

Comment:
not sure about writing this as HTTPClient - maybe with a space, so "HTTP client"?